src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -131,6 +145,13 @@ function checkTheRoleThatApplies(instrNum) {
 	var instrRole = $("#accessControlInfoForInstr" + instrNum + " div div p span").html();
 	$("input[id='instructorroleforinstructor" + instrNum + "']").filter("[value='" + instrRole + "']").prop("checked", true);
 	if (instrRole === "Custom") {
+		// Save original values of Custom Role
+		instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum] = [];
+		for (var i = 0; i < instructorPrivilegeValues.length; i++) {
+			var checkValue = $("#tunePermissionsDivForInstructor" + instrNum + " input[name='" + instructorPrivilegeValues[i] + "']").prop("checked");
+			instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum][instructorPrivilegeValues[i]] = checkValue;
```

Comment:
<code>instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum]</code> is an array, but you are using <code>instructorPrivilegeValues[i]</code>,which is a string, as a key. 

Maybe it will be better if you made <code>instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum]</code> an object instead.